### PR TITLE
VEN-259 UX: when saving/creating/removing product properties it should redirect back to product properties, not product edit

### PR DESCRIPTION
**Pull request by aleksandarpetrushev**
https://getvendo.atlassian.net/browse/VEN-259

### CODE REVIEW

**Comment by aleksandarpetrushev** [comment]
@damianlegawiec Previously we have made both product property fields required. Now consider this:
1. Go to product properties
2. Update existing property field
3. click Update - you are prompted to enter new property name (because the field is required)

What we can do is not make the fields required, but on update, make validation show an error that the attribute can not be blank. Should we do this?

File: app/controllers/spree/admin/products_controller.rb [hunk 1]
```diff
@@ -102,7 +102,11 @@ def find_resource
       end
 
       def location_after_save
-        spree.edit_admin_product_url(@product)
```
Comment (damianlegawiec) [inline app/controllers/spree/admin/products_controller.rb:105]: @aleksandarpetrushev we need a simple test for this

File: spec/features/admin/products/edit/properties_spec.rb [hunk 2]
```diff
@@ -17,6 +18,7 @@
       fill_in 'product_product_properties_attributes_0_value', with: 'Leather'
       click_button 'Update'
 
+      expect(page).to have_current_path(spree.admin_product_product_properties_path(product))
       within('#tabs') { click_link 'Properties' }
```
Comment (damianlegawiec) [inline spec/features/admin/products/edit/properties_spec.rb:22]:
```suggestion
```

File: spec/features/admin/products/edit/properties_spec.rb [hunk 3]
```diff
@@ -34,6 +36,7 @@
       find(:css, "#product_product_properties_attributes_0_show_property").set(false)
       click_button 'Update'
 
+      expect(page).to have_current_path(spree.admin_product_product_properties_path(product))
       within('#tabs') { click_link 'Properties' }
```
Comment (damianlegawiec) [inline spec/features/admin/products/edit/properties_spec.rb:40]:
```suggestion
```

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**